.travis.yml

```diff
@@ -2,7 +2,7 @@ language: node_js
 node_js:
 - 6.10.3
 - '7'
```

Comment:
we can remove 7, it'll never be on AWS.

we could also add `'10'` just to see if there's anything bad (eventually) coming down the pipe